### PR TITLE
Update pom.xml

### PR DESCRIPTION
**Pull request by nancycherian**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.thinknyx.web</groupId>
+  <groupId>com.google.com</groupId>
```
Comment (sinhadhirendra) [inline pom.xml]: change the url with vodafone

**Review by sinhadhirendra — COMMENTED** [review]
comment added

**Review by sinhadhirendra — APPROVED** [review]
all are good .proceed with merging

**Comment by nancycherian** [comment]
Completed the review comments

**Pull request merged**